joplin/templates/wagtailadmin/pages/listing/_table_headers_search.html

```diff
@@ -0,0 +1,81 @@
+{% load i18n %}
+
+{% comment %}
+
+Table headers for the page listing, when in 'explore' mode. Expects the following variables:
+
+sortable: if true, headings are links to wagtailadmin_explore with sort parameters applied.
+show_ordering_column: if true, an 'ordering' column is added.
+orderable: if true, the 'ordering' column is populated (again with links to wagtailadmin_explore).
+
+If either sortable or orderable is true, the following variables are also required:
+
+parent_page: The page instance currently being browsed (used to generate the correct wagtailadmin_explore urls)
+ordering: the current sort parameter
+
+Joplin Note: this file was modified from the original wagtail template found here: https://github.com/wagtail/wagtail/blob/23a89774adef4f609e33eb3feece90d6535588ee/wagtail/admin/templates/wagtailadmin/pages/listing/_table_headers_explore.html
+
+{% endcomment %}
+
+<tr class="table-headers">
+    {% if show_ordering_column %}
+        <th class="ord">
+            {% if orderable %}
+                {% if ordering == "ord" %}
+                    <a href="?{{ request.GET.urlencode }}" class="icon icon-order text-replace" title="{% trans 'Disable ordering of child pages' %}">{% trans 'Order' %}</a>
+                {% else %}
+                    <a href="?{{ request.GET.urlencode }}&ordering=ord" class="icon icon-order text-replace" title="{% trans 'Enable ordering of child pages' %}">{% trans 'Order' %}</a>
```

Comment:
btw @robbobfrh84 , i think this is where your sorting queries kept getting tagged on